cloudify_types/cloudify_types/component/operations.py

```diff
@@ -81,6 +81,7 @@ def upload_blueprint(**kwargs):
     blueprint_id = blueprint.get('id') or ctx.instance.id
     blueprint_archive = blueprint.get('blueprint_archive')
     blueprint_file_name = blueprint.get('main_file_name')
+    blueprint_labels = blueprint.get('labels', [])
```

Comment:
`blueprint_labels` is something different than `labels` (or deployment labels as we sometimes call them) https://docs.cloudify.co/latest/developer/blueprints/spec-blueprint-labels/ and https://docs.cloudify.co/latest/developer/blueprints/spec-labels/.  Just make sure it's really `labels` you're after.